docs/contributors/roadmap.md

```diff
@@ -1,48 +1,33 @@
-* Becoming the default local-cluster setup for Kubernetes
-
-## Non-Goals
-* Simplifying Kubernetes production deployment experience. Kube-deploy is attempting to tackle this problem.
-* Supporting all possible deployment configurations of Kubernetes like various types of storage, networking, etc.
-
-## Priorities
-This section contains the overall priorities of the minikube project, in rough order.
-
- * Setting up a well-tested, secure and complete Kubernetes cluster locally.
- * Cross Platform support (macOS, Linux, Windows)
- * Supporting existing Kubernetes features:
-    * Load Balancer support.
-    * Persistent disks.
- * Keeping up with new Kubernetes releases and features.
- * Development-focused features like:
-   * Mounting host directories.
-   * VPN/proxy networking.
- * Native hypervisor integration.
- * Support for alternative Kubernetes runtimes, like rkt.
- * Removing the VirtualBox dependency and replacing it with Hypervisor.framework/Hyper-V.
-
-## Timelines
-Minikube will release much faster than this, so this section is fairly speculative.
-This section is subject to change based on feedback and staffing.
-
-### Q1 2017
-
-* Release Kubernetes 1.6.0 alpha and beta releases packaged with minikube
-* Release Kubernetes 1.6.0 packaged with minikube within two days of GA upstream build
-* Run local e2e Kubernetes tests with minikube
-* Minikube no longer depends on libmachine
-* Minikube no longer depends on existing KVM driver
-* Native drivers are made default and packaged with minikube
-* Improve minikube start time by 30%
-* Add a no-vm driver for linux CI environments
+# minikube roadmap (2019)
+
+This roadmap is a living document outlining the major technical improvements which we would like to see in minikube during 2019. Please send a PR to suggest any improvements to it.
+
+## (#1) Make minikube the easiest way for developers to learn Kubernetes
+
+- Single step installation
+  - Users should not have to seperately install supporting binaries
+- Creation of a user-centric minikube website for installation & documentation
+
+## (#2) Diversify the minikube community
+
+- Add documentation for new minikube contributors
+- Ensure that all decisions are ratified publically by the minikube community
+
+## (#3) Make minikube robust and debuggable
+
+- Add pre-flight error checks for common connectivity issues 
+- Add pre-flight error checks for common configuration errors
+- Mark features & options not covered by continuous integration as `experimental`
+- Improve the `status` command so that it can diagnose common environmental issues
+- Make minikube usable offline
+
+## (#4) Official multi-node/multi-cluster support
+
+- Rebrand profiles as multi-cluster support
+- Integrate KIND for multi-node support
+- Add commands to add/remove nodes within an existing cluster
+
+## (#5) Improve minikube performance
+
+- Add support for lighter-weight deployment methods, such as container-based (LXD, Docker) or chroot
```

Comment:
This seems to contradict previous decisions, like dropping boot2docker and localkube for "standard" solutions ?

It will be hard for `minikube` to compete with vendor solutions, such as microk8s (for LXD) or Docker Desktop.

Reply:
I've adjusted this a bit. I don't believe for instance, that chroot or LXD support needs to contradict previous decisions. 

minikube already competes/cooperates with other vendor solutions. This is more to address the use cases where folks want to use minikube, but can't because it conflicts with other requirements. For instance, many corporate environments (like my own!) forbid VM's, and running VM's on Chromebooks is awkward at best. These users should still be able to use minikube, at least on Linux.

People already use "none" to work around the fact that minikube has nothing better to offer them, and it scares the hell out me. We can do better.